### DISCUSSION TITLE
What is nu's equivalent of `$0` (path to the current script)

### DISCUSSION THREAD

**Discussion by KSXGitHub** [nushell/nushell#7269] [Q&A]
I want to do `cd $(dirname $0)` in nu script.

I have tried to read the nu book but found nothing so far. If you can give me a link to similar variables, I'd appreciate.

**Comment by kubouch** [comment]
Try the `FILE_PWD` environment variable. (It might not work if you run `nu script.nu` but that's a bug and planned to be fixed.)

**Reply by KSXGitHub** [reply]
Unfortunately, `script.nu` is exactly my use case.
